package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfi-map-web",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
I would say this is more a minor update as being able to change layers is a new feature rather than a patch/fix.  But can see the argument for patch.  Happy either way, just my view.